static/src/javascripts/projects/common/modules/discussion/discussion-frontend.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+
+import config from 'lib/config';
+import fastdom from 'fastdom';
+import { integerCommas } from 'lib/formatters';
+import { loadScript } from 'lib/load-script';
+import mediator from 'lib/mediator';
+import reportError from 'lib/report-error';
+import { Loader } from './loader';
+
+const loadDiscussionFrontend = (loader: Loader, opts: Object) => {
+    const updateCommentCount = (element: HTMLElement, value: string): void => {
+        const formatted = integerCommas(value);
+
+        if (formatted) {
+            fastdom.write(() => {
+                element.textContent = formatted;
+            });
+        }
+    };
+
+    const onDiscussionFrontendLoad = (emitter): void => {
+        emitter.on('error', (feature, error) => {
+            reportError(error, { feature: `discussion-${feature}` }, false);
+        });
+
+        emitter.once('comment-count', (value: string): void => {
+            if (value === 0) {
+                loader.setState('empty');
+            } else {
+                /* By the time discussion frontent loads, the number of comments
+                   might have changed. If there are other comment counts element
+                   in the page refresh their value. */
+                const otherValues = document.getElementsByClassName(
+                    'js_commentcount_actualvalue'
+                );
+
+                if (otherValues.length) {
```

Comment:
I don't think you need the `otherValues.length` check here, `[...otherValues]` will equal `[]`  if `otherValues.length` is 0, so the forEach won't run.